frontend/src/components/core/Button.tsx

```diff
@@ -270,3 +270,26 @@ export const InlineShowIcon = styled.i.attrs(() => ({
   font-size: ${({ theme }) => theme.fontSize.medium};
   color: ${({ theme }) => theme.colors.font};
 `;
+
+export const SpectatorBackIcon = styled.i.attrs(() => ({
+  className: 'material-icons',
+}))`
+  position: absolute;
```

Comment:
We do have multiple icons, so I think I'll create a separate folder for those. This is really only used on this page, but it would be helpful to have those icons centralized so I'll make that change.

Reply:
Done ✅